cluster/ubuntu/util.sh

```diff
@@ -456,14 +590,14 @@ function prepare-push() {
       sleep 3
     fi
   else
-    # Run build.sh to get the required release 
+    # Run download-release.sh to get the required release 
     export KUBE_VERSION
-    "${KUBE_ROOT}/cluster/ubuntu/build.sh"
+    ${KUBE_ROOT}/cluster/ubuntu/download-release.sh
   fi
 }
 
 # Update a kubernetes master with expected release
-function push-master {
+function push-master() {
   source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE-"config-default.sh"}"
```

Comment:
I see two problems here.  First, it looks like nested quoting is intended here --- but is not done correctly.  BTW, the inner quoting is not needed.  Second, it looks like variable expansion with a default value is intended here, but it is also not done correctly --- the syntax for defaulting is :- not -.

Here is what `man bash` has to say about expansion with a default value:
```
       ${parameter:-word}
              Use Default Values.  If parameter is unset or null,  the  expan‐
              sion  of word is substituted.  Otherwise, the value of parameter
              is substituted.
```
All of config-default.sh is one word, in bash syntax.